packages/fast-jss-manager-angular/package.json

```diff
@@ -50,10 +50,13 @@
     "core-js": "^2.4.1",
     "jest": "^22.3.0",
     "jest-preset-angular": "^5.2.1",
+    "jss": "^9.8.1",
+    "jss-preset-default": "^4.3.0",
     "rxjs": "^5.5.6",
     "ts-jest": "^22.0.4",
     "tslint": "^5.9.1",
     "typescript": "^2.7.1",
+    "uuid": "^3.2.1",
```

Comment:
there are two different uuid versions here, can we consolidate to one?

Reply:
+1

Reply:
sure - standardized.